lib/build/bundle.js

```diff
@@ -220,15 +244,25 @@ exports.Bundle = class {
       if (includeBundles) {
         bundleMetadata[currentBundle.moduleId] = currentBundle.getBundledModuleIds();
       }
-
-      config.paths[currentBundle.moduleId] = location + '/' + currentBundle.moduleId;
+      //If build revisions are enabled, append the revision hash to the appropriate module id
+      config.paths[currentBundle.moduleId] = location + '/' + currentBundle.moduleId + (buildOptions.rev && currentBundle.hash ? '-' + currentBundle.hash : '');
     }
 
     if (includeBundles) {
       config.bundles = bundleMetadata;
     }
 
-    return 'requirejs.config(' + JSON.stringify(config) + ');'
+    return 'requirejs.config(' + JSON.stringify(config) + ')'
+  }
+  
+  setIndexFileConfigTarget(platform, location){
+    //Replace the reference to the vendor bundle in the "index.html" file to use the correct build revision file (or remove the build revision hash);
+    let indexLocation = platform.index;
+    let data = fs.readFileSync(indexLocation);
+    let outputDir = platform.baseUrl || platform.output; //If we have a baseUrl, then the files are served from there, else it's the output
+    let configMatcher = new RegExp(`\\b(?:src=("|')(.*))(${outputDir}(\/|\\\\)${this.moduleId}(.*?).js)(?:("|'))`);
```

Comment:
It would be nice with a warning if it was unable to update the index src path. Maybe the matcher doesn't find a vendor bundle path to replace.